### ISSUE TITLE
Failed to get the response content when using requests.post 

### ISSUE THREAD

**Issue by loctar665** [NCAR/rda-apps-clients#12]
Hi, I`m trying to download the ds084.1 data by rdams_client.py, the response code is 200, while  the content of response is empty. I got the sample control_file via rc.get_control_file_template(dsid), and the **template_dict** via rc.read_control_file(template). When i used rc.submit_json(template_dict) to got the response, it occured **json.decoder.JSONDecoderError: Expecting value: line 1 column 1 (char 0)**, the status code of response via requests.post was 200, but the content of response is **b' '**. I would like to consult you on a solution to this situation, I`m looking forward the response from you, thanks in advance.

**Comment by rpconroy** [comment]
Could you send the contents of 'template_dict' in your example? 
Or, send the full the code so I can try to reproduce your error.

**Comment by loctar665** [comment]
Thanks for your response, here is my full code.

”“”python

from rda_apps_clients import rdams_client
import json
import os
import requests

ds = 'ds084.1'
user_auth = rdams_client.get_cookies(username=' ',password=' ')

control_file = rdams_client.get_control_file_template(ds)
query_str = control_file['result']['template']
query_list = query_str.split('\n')
query_list[1] = 'date=201712310000/to/202101020000'
query_list[3] = 'param=TMP/R H/O3MR/U GRD/V GRD/PRES/HPBL'
query_list[4] = 'level=ISBL:1000'
query_list[6] = 'nlat=43'
query_list[7] = 'slat=33'
query_list[8] = 'wlon=124'
query_list[9] = 'elon=132'
product = ['product=Analysis']
for i in range(3,193,3):
    product.append('/{}-hour Forecast'.format(i))
query_list[10] = ''.join(product)
new_query = query_list[0]
for i in range(1,len(query_list)):
    new_query = new_query+'\n'+ query_list[i]

new_dic = rdams_client.read_control_file(new_query)
submit = rdams_client.submit_json(new_dic)

user_auth = rdams_client.get_authentication()
ret = requests.post(url,data=new_dic,auth=user_auth)
ret.content

“”“

**Comment by rpconroy** [comment]
For me, everything appears correct and seems to work through your call to call submit_json. The 'submit' variable returns a the correct response and provides a request index. 

The last two lines have problems for a number of reasons. 
1) `url` is not defined
2) `data=new_dic`: new_dic is a dictionary when it should be a json encoded string
3) `auth=user_auth`: we do not use basic authentication, rather, we use encoded cookies for this transaction. 

The bottom line is that if you're using the rdams_client code, you probably do not want or need to call `request.post` yourself. Are you getting something valid from the `submit` variable? Is that also empty?
